docs/_static/stravalib.css

```diff
@@ -0,0 +1,48 @@
+html, body {
+    font-size: 1.02rem;
+}
+
+body p {
+    color: #222;
```

Comment:
Maybe we should not hard-code colors here? These seem to lead to readability issues in dark mode. Maybe it's better (and easier to maintain) to let the theme decide about colors and/or font sizes to prevent any potential accessibility issues?